topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.md

```diff
@@ -0,0 +1,381 @@
+- Manage sequencing files (ab1, FASTQ, FASTA, FASTQ.GZ)
+- Clean sequences in an automated and reproducible manner
+- Perform alignments for each sequence
+- Have the necessary sequence format to submit to ENA
+- Submit raw reads to ENA using the ENA upload Tool
+time_estimation: 2h
+key_points:
+- Clean raw ab1 sequences and compare filtered sequences to NCBI nucleotidic database
+- Submit cleaned and unique sequences to European Nucleotide Archive (ENA) resource
+contributions:
+    authorship:
+        - Najatamk
+        - yvanlebras
+    testing:
+        - PaulineSGN
+        - yvanlebras
+    funding:
+        - pndb
+
+subtopic: ecologymetadatamgt
+---
+
+
+This tutorial will guide you through the necessary steps to manage and prepare sequencing files (ab1, FASTQ, FASTA) for submission to the genomic database ENA.
+This workflow will take you from raw sequences in AB1 format through all the necessary steps to integrate these sequences into the ENA genomic database. We will convert the files into FASTQ and FASTA formats after performing quality control.
+Additionally, we will perform alignments with the NCBI database to ensure the accuracy of your sequences.You will then need to fill a metadata Excel template to use the ENA upload Tool.
+The worklow is made of 17 Galaxy tools, we will  present them and explain what they do.
+The goal is to present an accessible and reproductible workflow for data submission.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Prepare raw data
+
+> <hands-on-title> Data Upload </hands-on-title>
+> 1. **Create a new history** for this tutorial
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. **Import** the raw sequences files.
+>
+>    ```
+>    https://data.indores.fr/api/access/datafile/3673
+>    https://data.indores.fr/api/access/datafile/3609
+>    ```
+>
+> 
+> 
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. **Rename** {% icon galaxy-pencil %} your datafiles
+>    - `3673` becomes `A2_RC_8F2_B.pl_HCOI.ab1`
+>    - `3609` becomes `A12_RC_9G4_B.md_HCOI.ab1`
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+> 4. **Check the datatype**
+>    - Make sure it is `ab1`, and change it if not.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md %}
+>
+> 5. **Build a Collection** containing these two files
+>
+>    {% snippet faqs/galaxy/collections_build_list.md %}
+>
+{: .hands_on}
+
+
+## Tools used in the "Prepare Data submission" Workflow
+
+Following steps take as input ab1 sequences files and produce filtered FastQ and Fasta files so sequences passing the quality checks are compared to NCBI nucleotidic database using Blastn operation.
+
+### Converting Ab1 files to FASTQ
+
+> <hands-on-title> ab1 to FASTQ converter </hands-on-title>
+> 1. {% tool [ab1 to FASTQ converter](toolshed.g2.bx.psu.edu/repos/ecology/ab1_fastq_converter/ab1_fastq_converter/1.20.0) %} on the ab1 files:

```

Comment:
final remark: instead of instructions like these "tool on these files", please use the standard format of really mentioning the parameter name and inputs explicitly, e.g.

```suggestion
> 1. {% tool [ab1 to FASTQ converter](toolshed.g2.bx.psu.edu/repos/ecology/ab1_fastq_converter/ab1_fastq_converter/1.20.0) %} with the following parameters:
>    - {% icon param-file %} *"Input ab1 file"*: `A2_RC_8F2_B.pl_HCOI.ab1`
> 2. Repeat this step for the file `A12_RC_9G4_B.md_HCOI.ab1`
>
```